### PR TITLE
Show python segment when Jupyter Notebook files are present

### PR DESCRIPTION
**Pull request by akselkvitberg**
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### Description

Adds a check for .ipynb files.
Jupyter Notebook is a popular tool that is often managed through anaconda.
Therefore it would be nice to show the python segment when a notebook file is present.

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
@Flueworks can you rebase on the latest main? I'll merge right after that. Looks 👍

**Review by JanDeDobbeleer — APPROVED** [review]

**Comment by akselkvitberg** [comment]
Done 👍

**Comment by JanDeDobbeleer** [comment]
> Done 👍

One more time, you're next (I'll wait myself). Sorry about that, there's a lot more interest in collaboration than I'd anticipated.

**Pull request merged**
